src/verify.c

```diff
@@ -0,0 +1,131 @@
+#include <openssl/bio.h>
+#include <openssl/cms.h>
+#include <openssl/err.h>
+#include <openssl/pem.h>
+#include <openssl/safestack.h>
+#include <openssl/x509.h>
+#include <openssl/x509_vfy.h>
+
+#include "./verify.h"
+
+return_t VF_init() {
+  ERR_load_crypto_strings();
+  OpenSSL_add_all_algorithms();
+  // Since none of these functions return useful error messages, per the
+  // openssl wiki documentation, we're just going to return success status.
+  // If this changes in future, we have the option to start failing
```

Comment:
True, but the return value is ignored.  For consistency, it's probably best to return void from this function, or to handle a potential failure in init().

Reply:
done